src/declarations/stencil-public-compiler.ts

```diff
@@ -1845,6 +1845,27 @@ export interface OutputTargetDist extends OutputTargetBase {
   dir?: string;
 
   collectionDir?: string | null;
+  /**
+   * When `true` this flag will transpile aliased import paths defined in
+   * a project's `tsconfig.json` to relative import paths in the compiled output's
+   * `dist-collection` bundle if it is generated (i.e. `collectionDir` is set).
+   *
+   * Paths will be left in aliased format if `false` or `undefined`.
+   *
+   * // tsconfig.json
+   * {
+   *   paths: {
+   *     "@utils/*": ['/src/utils/*']
+   *   }
+   * }
+   *
+   * // Source file
+   * import * as dateUtils from '@utils/date-utils';
+   * // Output file
+   * import { dateUtils } from '../utils/date-utils';
+   */
+  transpileAliasedImportPathsInCollection?: boolean | null;
```

Comment:
I'm not sure how I feel about the approach needed here to pass this config flag on to the `dist-collection` target. The `dist` output target creates a `dist-collection` output automatically. I'm still confused and concerned about how these targets play together (and appear to not be able to be opted out of)

Reply:
Yeah - IMO for now we need to put this on both `OutputTargetDistCollection` and  `OutputTargetDist`. Otherwise the only way to configure the former without explicitly declaring the latter.

As a consquence of that, we'd have to add validation logic to `validate-collection.ts` as well